xgcm/test/test_grid.py

```diff
@@ -104,7 +104,7 @@ def _assert_axes_equal(ax1, ax2):
     for pos, coord in ax1.coords.items():
         assert pos in ax2.coords
         assert coord == ax2.coords[pos]
-    assert ax1._periodic == ax2._periodic
+    assert ax1.periodic == ax2.periodic
```

Comment:
I wonder if this change is small and obvious enough to justify me changing tests? @TomNicholas, do you have opinions?